pkg/swagger/swagger.go

```diff
@@ -73,16 +93,18 @@ func Run(api, outputDir string) error {
 
 	xmsEnumList := map[string]struct{}{}
 
-	if api != "github.com/Azure/ARO-RP/pkg/api/v20200430" {
+	names := []string{"OpenShiftClusterList", "OpenShiftClusterCredentials"}
```

Comment:
I think a lot of the code around here could be more neatly parameterised and defined in g